lib/go/edgecontext/edgecontext.go

```diff
@@ -24,6 +25,13 @@ const LoIDPrefix = "t2_"
 // does not have the correct prefix.
 var ErrLoIDWrongPrefix = errors.New("edgecontext: loid should have " + LoIDPrefix + " prefix")
 
+// Locale codes should contain either a language, or a language and region specifier
+// separated by an underscore.
+// e.g. en, en_US
+var LocaleRegex = regexp.MustCompile(`^[a-z]{2}(_[A-Z]{2})?$`)
+
+var ErrInvalidLocaleCode = errors.New("edgecontext: locale code should match format: en, en_US")
```

Comment:
style nit: this should be in a `var` block with the other error and should include a docstring comment:

```go
var (
  // ErrLoIDWrongPrefix is an error could be returned by New() when passed in LoID
  // does not have the correct prefix.
  ErrLoIDWrongPrefix = errors.New("edgecontext: loid should have " + LoIDPrefix + " prefix")

  // ErrInvalidLocaleCode should have a comment
  ErrInvalidLocaleCode = errors.New("edgecontext: locale code should match format: en, en_US")
)
```

Reply:
👍  Updated in https://github.com/reddit/edgecontext/pull/12/commits/ca3ecb78969b7173ea01c490af773764ed983626